operations/iuf/workflows/management_rollout.md

```diff
@@ -21,6 +21,15 @@ Once this step has completed:
 
 - Host firmware has been updated on management nodes
 
+Note: Before proceeding with `management-nodes-rollout` iSCSI SBPS configuration has to be done where worker nodes need
+to be configured as iSCSI targets for iSCSI SBPS. Unlike in CSM 1.6.0, where all worker nodes were configured as iSCSI
+targets by default without requiring any manual steps, in CSM 1.7.0, selective worker node personalization is introduced
+where it requires to create an HSM group named `iscsi_worker` and add the xnames of worker nodes which are intended to
+be configured as iSCSI targets. If this is not done, none of the worker nodes will be configured as iSCSI targets. So
```

Comment:
Like above, this isn't true, right? If the group isn't created, it means every worker will be configured for iSCSI, unless something changed since we talked about this last.